pages/faq.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Section from 'common/components/Section/Section';
+import Question from 'components/faq/question';
+import QuestionAnswerData from 'components/faq/questions';
+import styles from './styles/faq.css';
+
+const FAQ = () => (
+    <Section title="General Questions" theme="white">
+      <Section theme="white" headingLines={false}>
+        <div className={styles.container}>
+          {QuestionAnswerData().general.map(faq => (
```

Comment:
If you change the `questions` to an object, you'll need to drop the `()` from each reference on here

Reply:
will do